sklearn/metrics/_scorer.py

```diff
@@ -652,6 +653,8 @@ def make_scorer(score_func, *, greater_is_better=True, needs_proba=False,
 neg_mean_gamma_deviance_scorer = make_scorer(
     mean_gamma_deviance, greater_is_better=False
 )
+neg_pinball_loss_scorer = make_scorer(pinball_loss,
+                                      greater_is_better=False)
```

Comment:
Actually now I realize that this is useless, because the default value of alpha is `0.50` which means that this is  the same as the (2 times) `neg_mean_absolute_error_scorer`. So instead we could either introduce a range of scorers for different common values of alpha (e.g. `neg_05p_pinball_loss_scorer`, `neg_25p_pinball_loss_scorer`, `neg_50p_pinball_loss_scorer`, `neg_75p_pinball_loss_scorer`, `neg_95p_pinball_loss_scorer`...).

Or alternatively we could document how to build you own scorer with a snippet in the user guide, for instance in conjunction with the `scoring` parameter of `cross_val_score`.

Reply:
I would vote in favor of a snippet. You can compare a linear regression and a decision tree using L1, L2 but what about pinball(., alpha=0.95). I wonder.

Reply:
I pushed such an example code snippet in 0fa7e10. We still need to revert the generic `neg_pinball_loss_scorer`.